.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**Steps To Reproduce**
+
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+
+A clear and concise description of what you expected to happen.
+
+**Logs**
+
+If applicable, provide the link to the log files that contain the issue.
+
+**Screenshots**
+
+If applicable, add screenshots to help explain your problem.
+
+
+**Environment:**
+
+If applicable, add the information about that you are running and their versions
+
+
+**Additional context**
+
+If possible, provide the failing test the reveals the issue.
```

Comment:
provide _a_ failing test; maybe link to a guide describing now to create an acceptance test?

Reply:
do you have such a guide?

Reply:
no, but we can create one :)